src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -26,5 +27,147 @@ public partial class AppDomain
                 AppContext.UnhandledException -= value;  
             }  
         }  
+
+        public string DynamicDirectory => null;
+        public string FriendlyName => "DefaultDomain";
+        public int Id => DefaultADID;
+        public bool IsFullyTrusted => true;
+        public bool IsHomogenous => true;
+        public bool ShadowCopyFiles => false;
+        public event System.EventHandler DomainUnload
+        {
+            add  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+  
+            remove  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+        }
+
+        public event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException
+        {  
+            add  
+            {  
+                AppDomainAugments.FirstChanceException += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.FirstChanceException -= value;  
+            }  
+        }  
+
+        public event System.EventHandler ProcessExit
+        {  
+            add  
+            {  
+                AppDomainAugments.ProcessExit += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.ProcessExit -= value;  
+            }  
+        }  
+
+        public void AppendPrivatePath(string path) { throw new PlatformNotSupportedException(); }
+        public string ApplyPolicy(string assemblyName) => AppDomainAugments.ApplyPolicy(assemblyName);
+        public void ClearPrivatePath() { throw new PlatformNotSupportedException(); }
+        public void ClearShadowCopyPath() { throw new PlatformNotSupportedException(); }
+        public static System.AppDomain CreateDomain(string friendlyName) { throw new PlatformNotSupportedException(); }
+        public int ExecuteAssembly(string assemblyFile) => ExecuteAssembly(assemblyFile, null, null, System.Configuration.Assemblies.AssemblyHashAlgorithm.None);
+        public int ExecuteAssembly(string assemblyFile, string[] args) => ExecuteAssembly(assemblyFile, args, null, System.Configuration.Assemblies.AssemblyHashAlgorithm.None);
+        public int ExecuteAssembly(string assemblyFile, string[] args, byte[] hashValue, System.Configuration.Assemblies.AssemblyHashAlgorithm hashAlgorithm)
```

Comment:
All the ExecuteAssembly methods should be just implemented here - no need to have it in runtime. Something like:

```
var a = Assembly.LoadFrom(path);
var entry = a.EntryPoint;
if (entry.GetParameters().Length > 0)
{
    result = entry.Invoke(null, new object[] { args });
}
else
{
    result = entry.Invoke(null);
}
```